### PR TITLE
Update for version 6.0

### PR DESCRIPTION
**Pull request by briacp**

### CODE REVIEW

**Comment by brandelune** [comment]
The macOS selector is not fixed. Clicking on "[Standard Version - OmegaT 6.0.0](https://omegat.org/download#standard_osx_signed)" looks for the signed build and stalls. Same for the "Download" button.

Screen Shot 2023-06-25 at 22 31 53

**Comment by briacp** [comment]
> The macOS selector is not fixed.

It should be OK now

**Comment by brandelune** [comment]
Sorry to be picky, but could you make it a way that the sentence "If this version is not available, you can try to download the unsigned version." is not displayed?

Screen Shot 2023-06-26 at 5 08 23

It is confusing since the download *is* the unsigned version.

**Comment by brandelune** [comment]
I think we should be good with https://github.com/omegat-org/omegat-website/pull/89. Could you check?

**Comment by briacp** [comment]
> I think we should be good with #89. Could you check?

It's OK, I merged #89

**Review by brandelune — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -43,8 +43,6 @@ social:
     - https://twitter.com/OmegaTOfficial
     - https://www.facebook.com/OmegaTOfficial
 
-google_analytics: UA-87327-16
-
 # i18n plugin, "en" is listed twice to force jekyll-multiple-languages-plugin
 # to create a "en/" folder, like the original OmegaT website.
 languages: [ "en" , "fr" ,"ar", "ca", "co", "cs", "de", "el", "es", "gl", "id", "ia", "it", "ko", "hu", "ja", "nl", "pl", "pt_BR", "pt", "ru", "sq", "sk", "si", "hr", "fi", "sv", "uk", "zh_CN", "zh_TW", "mfe", "en" ]
```
Comment (briacp) [inline _config.yml:48]:
```suggestion
languages: [ "en", "ar", "ca", "co", "cs", "de", "el", "es", "fi", "fr", "gl", "hr", "hu", "ia", "id", "it", "ja", "ko", "mfe", "nl", "pl", "pt", "pt_BR", "ru", "si", "sk", "sq", "sv", "uk", "zh_CN", "zh_TW", "en" ]
```
sort language codes

**Comment by briacp** [comment]
just noticed that this PR was setup as `master` -> `6.0` which wasn't making much sense. No big deal, all the modifications are live and good.

**Pull request merged**